pyproject.toml

```diff
@@ -50,15 +50,21 @@ proxy = [
     "pyyaml",
     "rq",
     "orjson",
+    "tenacity",
+    "cohere"
```

Comment:
why is cohere added as a dependency? We don't need it all requests are made via http requests